src/component/mxgraph/ShapeConfigurator.ts

```diff
@@ -13,17 +13,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { MxGraphFactoryService } from '../../service/MxGraphFactoryService';
+
 import { mxgraph } from 'ts-mxgraph';
 import { ShapeBpmnElementKind } from '../../model/bpmn/shape/ShapeBpmnElementKind';
 import { EndEventShape, StartEventShape, ThrowIntermediateEventShape, CatchIntermediateEventShape } from './shape/event-shapes';
 import { ExclusiveGatewayShape, ParallelGatewayShape, InclusiveGatewayShape } from './shape/gateway-shapes';
 import { ReceiveTaskShape, ServiceTaskShape, TaskShape, UserTaskShape } from './shape/task-shapes';
 
+declare const mxClient: typeof mxgraph.mxClient;
+declare const mxShape: typeof mxgraph.mxShape;
+declare const mxCellRenderer: typeof mxgraph.mxCellRenderer;
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
```suggestion
// TODO should be 'typeof mxgraph.mxSvgCanvas2D', current type definition does not declare 'minStrokeWidth'
// This will be managed when switching to 'mxgraph-type-definitions'
// eslint-disable-next-line @typescript-eslint/no-explicit-any
```